src/en/server-hosting/setting-up-redbot.md

```diff
@@ -0,0 +1,85 @@
+# Setting up your own discord bot for your server
+
+You may have noticed in the official Wizards Den discord server that we have a bot that provides some nice features such as linking GitHub issues/pull requests, OOC & Discord bridge and a status command. 
+
+Luckily for you, this is easy to set up with the help of the [Red Discord Bot](https://github.com/Cog-Creators/Red-DiscordBot/tree/V3/develop). A modular self-hosted Python discord bot. You do not need to know anything about programming to set up this bot. And this guide will help you with how to install and use our cogs on the bot.
+
+```admonish note
+We only provide support for the official wizard-cogs in the #hosting channel on the discord, Support for Red itself or other cogs is not provided by us, please get support in the official RedBot Discord or the appropriate cog developer.
+```
+
+## Setup
+1. Follow the [Red Documenation](https://docs.discord.red/en/stable/install_guides/index.html) on how to install and start the Base Red bot. Note that like an SS14 server. You need a computer that will stay turned on for your bot to function.
+2. If you have not already, give [Red's Getting started](https://docs.discord.red/en/stable/getting_started.html#getting-started) page a read.
+3. After you get your bot set up and invited to your server, you can now install the wizard-cogs repository to your bot. [p] in this case is your bots prefix you chose during redbot setup.
+```
+Enable the built-in downloader cog
+[p]load downloader
+
+Add the wizard-cogs repository
+[p]repo add wizard-cogs https://github.com/space-wizards/wizard-cogs
+
+Install the cog of choice, check out the GitHub link to see your options
+[p]cog install wizard-cogs <cog_name>
+```
+
+```admonish note title="Psss"
+Looking for other interesting cogs? Check out the [Redbot Cog index](https://index.discord.red/). Like stated above we don't provide support for these cogs.
+```
+```admonish note title="Psss Combo 2"
+Not an English server? Some cogs have translations, You can change this with ```[p]set locale <language_code>```. Our cogs don't support other languages than English currently.
+```
+Now you are ready, please be sure to check the [Wizard-Cogs Github](https://github.com/space-wizards/wizard-cogs) in case any new cogs have been added.
+
+
+Otherwise, I will list the most interesting ones here with setup instructions
+
+### Game server status
+![SS14 Status](https://github.com/space-wizards/wizard-cogs/raw/master/media/SS14-game-status-example.png)
```

Comment:
Copy all these images to the local repo so the links don't break in the future.

Reply:
My idea was that so we don't have to update the images here if we update them on the wizard cogs repo. But I can do when I got time